pages/express-relay/_meta.json

```diff
@@ -0,0 +1,38 @@
+{
+  "documentation-home": {
+    "title": "← Documentation Home",
+    "href": "/home"
+  },
+
+  "-- Express Relay": {
+    "title": "Express Relay",
+    "type": "separator"
+  },
+  "index": "Introduction",
+
+  "-- How-to Guides": {
+    "title": "How-To Guides",
+    "type": "separator"
+  },
+
+  "integrate-express-relay": "Integrate Express Relay",
```

Comment:
I suggest flattening this item out into two items in the nav, one that's "Integrate as a Searcher" and "Integrate as a Protocol". That's more discoverable for the reader, and the nav menu isn't very crowded so we don't need to nest much (imo)